docs/ci.md

```diff
@@ -12,10 +12,10 @@ supports.
 For this configuration, a new Secret should be created in the GitHub
 Settings of the repository. 'Normal' environment variables seem not possible.
 
-An example of the GitHub Secret that will build the container-images on AMD64,
-and both 32-bit and 64-bit Arm platforms:
+An example of the GitHub Variable that will build the container-images on
+AMD64, and both 32-bit and 64-bit Arm platforms:
 
-- `BUILD_PLATFORMS`: `linux/amd64,linux/arm64,linux/arm/v7`
```

Comment:
@nixpanic there are still more reference to secret in this file.

Reply:
aaaah!